ci.sh

```diff
@@ -72,56 +86,32 @@ install_prerequisite() {
       pixman-static \
       pkgconf \
       python3 \
+      xz \
       zlib-static
   fi
 }
 
-clone_qemu_repository() {
-  git clone \
-    --branch v6.0.0 \
-    --depth 1 --recurse-submodules \
-    https://github.com/qemu/qemu
-}
-
-patch_qemu_for_alpine() {
```

Comment:
Have you verified that this is not necessary anymore?

Reply:
technically it is not necessary since that system emulator was no longer compiled (only x86_64 and aarch64 are)

Reply:
it is no longer needed